skipper/cli.py

```diff
@@ -18,12 +19,32 @@
 DOCKER_TAG_FOR_CACHE = "cache"
 
 
+def _validate_publish(ctx, param, value):
+    # pylint: disable=unused-argument
+    if value:
+        matcher = compile_expression(r'^(\d+):(\d+)$')
```

Comment:
There is also an option to publish a range of ports, e.g.
`docker run -p 4000-6000:4000-6000`